ksql-engine/src/test/java/io/confluent/ksql/integration/IntegrationTestHarness.java

```diff
@@ -408,6 +482,9 @@ protected void after() {
     final Map<String, Object> config = clientConfig();
     config.put(ConsumerConfig.GROUP_ID_CONFIG, UUID.randomUUID().toString());
     config.put(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest");
+    // Try to keep consumer groups stable:
+    config.put(ConsumerConfig.HEARTBEAT_INTERVAL_MS_CONFIG, 10_000);
+    config.put(ConsumerConfig.SESSION_TIMEOUT_MS_CONFIG, 30_000);
```

Comment:
Add some settings to increase the default timeouts on groups, so that tests are less likely to run into rebalances.